src/PlanForm.tsx

```diff
@@ -53,50 +61,109 @@ export class PlanForm extends React.Component<PlanType, IState> {
-          </CardContent>
-          <CardActions disableSpacing>
-            <IconButton onClick={() => this.props.submitPlan(this.props.name)} edge="end" aria-label="comments">
-              <SendIcon />
-            </IconButton>
-          </CardActions>
-        </Card>
-      </Box>
-    );
+    if (this.props.name == ""){
+      return (
+        <Paper style={{height: "83vh", overflow: 'auto', margin: "auto"}}>
+          <Box>
+            <Card style={{height: "6vh"}} raised={true}>
+              <CardContent>
+                <Typography align="center" variant="h5" component="h1" gutterBottom>
+                  Plan Form
+                </Typography>
+              </CardContent>
+            </Card>
+          <Box height="2vh"></Box>
+            <Card raised={true}>
+              <CardHeader
+                avatar={
+                  <AccountCircleIcon fontSize='large' />
+                }
+                titleTypographyProps={{variant:'h6' }}
+                title={"Select a plan."}
+              />
+              <CardContent>
+                <Typography>
+                    Select a plan from the available plans list.
+                </Typography>
+              </CardContent>
+            </Card>
+          </Box>
+        </Paper>
+      );
+    } else {
+      return (
+        <Paper style={{height: "83vh", overflow: 'auto', margin: "auto"}}>
+          <Box>
+            <Card style={{height: "6vh"}} raised={true}>
+              <CardContent>
+                <Typography align="center" variant="h5" component="h1" gutterBottom>
+                  Plan Form
+                </Typography>
+              </CardContent>
+            </Card>
+          <Box height="2vh"></Box>
+            <Card raised={true}>
+              <CardHeader
+                avatar={
+                  <AccountCircleIcon fontSize='large' />
+                }
+                action={
+                  <IconButton aria-label="settings">
+                    <MoreVertIcon />
+                  </IconButton>
+                }
+                titleTypographyProps={{variant:'h6' }}
+                title={this.props.name}
+              />
+              <CardContent>
+                <Typography > 
+                    {this.props.allowedPlans.plans_allowed[this.props.name]["description"] ? 
+                    this.props.allowedPlans.plans_allowed[this.props.name]["description"] : "No plan description found."}
+                </Typography>
+              </CardContent>
+              <CardContent>
+                <Typography variant="body2" color="textSecondary" component="p">
+                  Enter the plan parameters!
+                </Typography>
+                <List>
+                  {this.props.allowedPlans.plans_allowed[this.props.name].parameters.map(
+                    (parameterObject: IParameter) => (
+                      <ListItem divider={true} button={true} key={parameterObject.name}>
```

Comment:
It looks better to me without dividers, the form feels odd to me highlighting the bigger horizontal box and then the actual entry box.